### ISSUE TITLE
Popup closes immediately in some cases

### ISSUE THREAD

**Issue by freebrowser1** [p0deje/Maccy#823]
### Before Submitting Your Bug Report

- [X] I have verified that there isn't already an issue reporting the same bug to prevent duplication.
- [X] I have seen the [FAQ](https://github.com/p0deje/Maccy?tab=readme-ov-file#faq).

### Maccy Version (see 'About' window)

1.0.0

### macOS Version

14.5

### Maccy Settings

```Shell
{
    "KeyboardShortcuts_delete" = "{\\"carbonModifiers\\":2048,\\"carbonKeyCode\\":51}";
    "KeyboardShortcuts_pin" = "{\\"carbonModifiers\\":2304,\\"carbonKeyCode\\":20}";
    "KeyboardShortcuts_popup" = "{\\"carbonModifiers\\":4352,\\"carbonKeyCode\\":18}";
    "LaunchAtLogin__hasMigrated" = 1;
    "NSWindow Frame com.sindresorhus.Settings.FrameAutosaveName" = "689 495 542 437 0 0 1920 1080 ";
    SUAutomaticallyUpdate = 0;
    SUEnableAutomaticChecks = 0;
    SUHasLaunchedBefore = 1;
    SUSendProfileInfo = 0;
    enabledPasteboardTypes =     (
        "public.file-url",
        "public.tiff",
        "public.png",
        "public.html",
        "public.rtf",
        "public.utf8-plain-text"
    );
    ignoredPasteboardTypes =     (
        "de.petermaurer.TransientPasteboardType",
        "com.typeit4me.clipping",
        "Pasteboard generator type",
        "com.agilebits.onepassword",
        "net.antelle.keeweb"
    );
    migrations =     {
        "2020-04-25-allow-custom-ignored-types" = 1;
        "2020-06-19-use-keyboardshortcuts" = 1;
        "2020-09-01-ignore-keeweb" = 1;
        "2021-02-20-allow-to-customize-supported-types" = 1;
        "2021-06-28-add-title-to-history-item" = 1;
        "2021-10-16-remove-dynamic-pasteboard-types" = 1;
        "2022-08-01-rename-suppress-clear-alert" = 1;
        "2022-11-14-add-html-rtf-to-supported-types" = 1;
        "2023-01-22-add-regexp-search-mode" = 1;
    };
}
```

### Description

When clicking the Hotkey (I use cmd+Ctrl+1) in some cases the popup opens, but immediately closes, so I have to click on another window and it works. In other cases it does work as intended.
These 'some cases' are:
* a Terminal (standard macOS terminal) when a password is asked for ssh login
* in a password field in a browser (Chrome, Firefox)

### Steps to Reproduce

1. Open Terminap.app
2. Connect to an ssh server by `ssh user@server`
3. Press Maccy hotkey when prompted for a password
4. Then it opens and closes again very quickly.

**Comment by p0deje** [comment]
That's a known issue when macOS prevents custom window from showing over password field. Try changing "Avoid taking application focus" in Maccy advanced settings. That's the only known workaround today.

**Comment by freebrowser1** [comment]
Thanks, that indeed seems to work !

**Comment by p0deje** [comment]
This is also fixed in 2.0 beta (though there are some other problems there to be fixed yet).
